### PR TITLE
remove unused code?

### PR DESCRIPTION
**Pull request by migoi**
## Description

this one could be remove i guess

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code is written according to the guidelines found [here] (https://yetiforce.com/en/developer-documentation/standards/167-how-to-create-php-files.html).
- [x] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/API/models/DAV.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static function runCronCardDav()
 			}
 		}
 		$cardDav = new API_CardDAV_Model();
-		$cardDav->log = $dav->log;
+		// $cardDav->log = $dav->log;  // fixme - seems to be old code - remove?
```
Comment (mariuszkrzaczkowski) [inline modules/API/models/DAV.php]: To remove

File: modules/Import/views/Main.php [hunk 2]
```diff
@@ -120,7 +120,7 @@ public static function showResult($importInfo, $importStatusCount)
 
 		$viewer = new Vtiger_Viewer();
 
-		$viewer->assign('SKIPPED_RECORDS', $skippedRecords);
+		// $viewer->assign('SKIPPED_RECORDS', $skippedRecords); // fixme: nowhere else in code
```
Comment (mariuszkrzaczkowski) [inline modules/Import/views/Main.php]: To remove

File: modules/API/models/DAV.php [hunk 3]
```diff
@@ -55,7 +55,7 @@ public static function runCronCalDav()
 			}
 		}
 		$cardDav = new API_CalDAV_Model();
-		$cardDav->log = $dav->log;
+		// $cardDav->log = $dav->log;  // fixme - seems to be old code - remove?
```
Comment (mariuszkrzaczkowski) [inline modules/API/models/DAV.php]: To remove

Comment (migoi) [inline modules/API/models/DAV.php]: removed

**Pull request merged**